media/robots.txt

```diff
@@ -3,11 +3,11 @@ Crawl-delay: 5
 Sitemap: https://developer.mozilla.org/sitemap.xml
 Request-rate: 1/5
 
-Disallow: /*admin*
+Disallow: /*/admin/*
```

Comment:
This should be ``Disallow: /admin/``, which will catch all Django admin URLs.  They don't get the funky MDN locale prefix.